src/test/java/com/google/ehub/servlets/DashboardServletTest.java

```diff
@@ -0,0 +1,129 @@
+package com.google.ehub.servlets;
+
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.ehub.data.EntertainmentItemDatastore;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@RunWith(JUnit4.class)
+public class DashboardServletTest {
+  private final DashboardServlet servlet = new DashboardServlet();
+  private final PrintWriter printWriter = new PrintWriter(new StringWriter());
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  private static final String SEARCH_VALUE_PARAMETER_KEY = "searchValue";
+  private static final String SORTING_DIRECTION_PARAMETER_KEY = "sortingDirection";
+  private static final String ASCENDING_PARAMETER_VALUE = "ASCENDING";
+  private static final String JSON_CONTENT_TYPE = "application/json";
+
+  private static final int MAX_SEARCH_VALUE_CHARS = 150;
```

Comment:
switch the order of the the static fields and instance fields

Reply:
OK I switcher order of static fields in last commit.